docs/11.1/api/core.md

```diff
@@ -2228,16 +2213,24 @@ any keyboard events.
 
 _core.updateData(data, [source])_
 
-Replaces the dataset with a new one. Unlike `setData` and the `loadData` methods, it doesn't reset the
-state of the table.
+The `updateData()` method:
+- Replaces Handsontable's [`data`](@/api/options.md#data)
+- Keeps cells' [configuration options](@/guides/getting-started/setting-options.md) unchanged
+- Keeps rows' and columns' states unchanged (e.g. [row height](@/guides/rows/row-height.md), [column width](@/guides/columns/column-width.md), frozen [rows](@/guides/rows/row-freezing.md) and [columns](@/guides/columns/column-freezing.md) etc.)
```

Comment:
Same notes as with `loadData` above.

`updateData` doesn't clear:
- index mappers
- cell meta

Otherwise it does the same thing as `loadData`.

Reply:
@jansiegel - please have a look now at the updated version: 64e6f14